server/src/main/kotlin/suwayomi/tachidesk/anime/model/table/AnimeTable.kt

```diff
@@ -48,7 +49,7 @@ fun AnimeTable.toDataClass(mangaEntry: ResultRow) =
         mangaEntry[artist],
         mangaEntry[author],
         mangaEntry[description],
-        mangaEntry[genre],
+        mangaEntry[genre]?.split(",")?.trimAll().orEmpty(),
```

Comment:
how about adding a `.toGenreList()` helper function for this?

Reply:
That function would be available to every String if we did it that way, I can do it though

Reply:
yeah, and make it `inline` and `internal`

Reply:
`inline` as well? It wont give any performance benefits :pepehmm:

Reply:
Done